File: samples/client/petstore/java/feign/src/main/java/org/openapitools/client/api/DefaultApi.java
package org.openapitools.client.api;

import org.openapitools.client.ApiClient;
import org.openapitools.client.EncodingUtils;
import org.openapitools.client.model.ApiResponse;

import org.openapitools.client.model.Dummy200Response;

import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;
import java.util.Map;
import feign.*;

@javax.annotation.Generated(value = "org.openapitools.codegen.languages.JavaClientCodegen")
public interface DefaultApi extends ApiClient.Api {


  /**
   * 
   * dummy
   * @return Dummy200Response
   */
  @RequestLine("GET /example/someMethod")
  @Headers({
    "Accept: */*",
  })
  Dummy200Response dummy();

  /**
   * 
   * Similar to <code>dummy</code> but it also returns the http response headers .
   * dummy
   * @return A ApiResponse that wraps the response boyd and the http headers.
   */
  @RequestLine("GET /example/someMethod")
  @Headers({
    "Accept: */*",
  })
  ApiResponse<Dummy200Response> dummyWithHttpInfo();


}

File: samples/server/petstore/kotlin-server/jaxrs-spec/src/main/kotlin/org/openapitools/server/apis/DefaultApi.kt
package org.openapitools.server.apis;

import org.openapitools.server.models.Dummy200Response
import org.openapitools.server.models.ExampleResponse

import javax.ws.rs.*
import javax.ws.rs.core.Response


import java.io.InputStream



@Path("/")
@javax.annotation.Generated(value = arrayOf("org.openapitools.codegen.languages.KotlinServerCodegen"))
class DefaultApi {

    @GET
    @Produces("*/*")
    suspend fun dummy(): Response {
        return Response.ok().entity("magic!").build();
    }
}
